app/Models/ComponentGroup.php

```diff
@@ -15,21 +15,38 @@
 use CachetHQ\Cachet\Models\Traits\SearchableTrait;
 use CachetHQ\Cachet\Models\Traits\SortableTrait;
 use CachetHQ\Cachet\Presenters\ComponentGroupPresenter;
+use Illuminate\Database\Eloquent\Builder;
 use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Collection;
 use McCool\LaravelAutoPresenter\HasPresenter;
 
 class ComponentGroup extends Model implements HasPresenter
 {
     use SearchableTrait, SortableTrait, ValidatingTrait;
 
     /**
+     * Viewable only logged in users.
+     *
+     * @var int
+     */
+    const VISIBLE_LOGGED_IN = 0;
```

Comment:
I feel like these two constants are weirdly named. `LOGGED_IN` and `PUBLIC` seem to be slightly mismatched.

`PUBLIC` + `PRIVATE`? `AUTHENTICATED` + `GUEST`?